### PR TITLE
#82 show instructions on scene start

### LINKED ISSUE #75: Add back to change number range

### LINKED ISSUE #82: Show instructions on scene start

### PR DESCRIPTION
**Pull request by juliakobusinska98**
- added instructions on scene start
- added ' back to range choice' button in  Numbers Scene
image

### CODE REVIEW

**Review by ani119 — COMMENTED** [review]
it is blocked by #81 add signs scene, but after merge #81 is will add aprove

**Review by AgaMazur — DISMISSED** [review]

**Review by przemytka — DISMISSED** [review]

**Comment by ani119** [comment]
There is an conflict at one file :(

**Review by ani119 — APPROVED** [review]

**Review by AgaMazur — APPROVED** [review]

**Review by przemytka — APPROVED** [review]

**Pull request merged**
